Library/Formula/cycript.rb

```diff
@@ -0,0 +1,91 @@
+class Cycript < Formula
+  desc "Modify apps using a hybrid of Objective-C++ and JavaScript syntax."
+  homepage "http://www.cycript.org"
+  url "git://git.saurik.com/cycript.git",
+    :tag => "v0.9.592",
+    :revision => "97bec96b43b66ab78af95d2b6c6c24f0d0a8006a"
+
+  head "git://git.saurik.com/cycript.git"
+
+  depends_on "bison" => :build
+  depends_on "android-sdk" => :build
+  depends_on "readline"
+
+  patch :DATA
+
+  def install
+    ENV.append "CPPFLAGS", "-Wno-inconsistent-missing-override"
```

Comment:
> I guess I should really disable -Werror if you are not in maintainer mode, […]

Yes, you should! `-Werror` is really only a developer tool and should never be used when distributing software. It is a good goal to eliminate all warnings during development and `-Werror` helps with this, but you never know what warnings there will be in a future or untested compiler. (Or maybe caused by one of the libraries you use.)